libs/components/forms/src/lib/modules/form-error/form-errors.component.html

```diff
@@ -1,104 +1,133 @@
-<ng-container *ngIf="labelText && showErrors">
-  <ng-container *ngIf="errors">
-    <sky-form-error
-      *ngIf="errors['required']"
-      errorName="required"
-      [errorText]="'skyux_form_error_required' | skyLibResources: labelText"
-    />
+@if (labelText && showErrors) {
+  <ng-container>
```

Comment:
These `ng-container`s are no longer needed. Containers don't exist in the DOM, they purely exist to attach structural directives to. In fact, when you put a structural directive on a real element, it actually gets wrapped in a container that is showing/hiding or replicating or whatever. The new control flow syntax just makes what's actually happening with the DOM manipulation more apparent.